lib/classes/add_printers_users.rb

```diff
@@ -0,0 +1,12 @@
+class AddPrintersUsers
+  def self.apply!
+    new.apply
+  end
```

Comment:
Don't think there's any need to instantiate anything in this case, you can put everything under `def self.apply!`

Reply:
Sure.. i'll add directly code in `self.apply!`.